Public/Sdk/Public/Managed/Shared/types.dsc

```diff
@@ -78,6 +78,11 @@ export interface Assembly extends Deployment.Deployable {
      * that conflict with other dependencies.
      */
     runtimeContentToSkip?: Deployment.DeployableItem[];
+
+    /**
+     * Similiar to runtimeContent, but specific to native files.
+     */
+    nativeContent?: Deployment.Definition;

```

Comment:
it looks like you don't need this any longer, so please remove it

Reply:
Had problems with the merge, but done. Thanks!